src/domain/synthetics/accountStats/useAccountVolumeStats.ts

```diff
@@ -0,0 +1,95 @@
+import { gql } from "@apollo/client";
+import { getSubsquidGraphClient } from "lib/subgraph";
+import { useMemo } from "react";
+import useSWR from "swr";
+import { subDays, format, eachDayOfInterval } from "date-fns";
+import { toUtcDayStart } from "lib/dates";
+import { CONFIG_UPDATE_INTERVAL } from "lib/timeConstants";
+import { ARBITRUM, AVALANCHE } from "config/chains";
+
+const LARGE_ACCOUNT_CHAINS = [ARBITRUM, AVALANCHE];
+
+export function useAccountVolumeStats(params: { account?: string }) {
```

Comment:
We already have useAccountStats and usePeriodAccountStats

as i see this hook request daily buckes, probably it make sense to rename this to something specific to this logic, because at first glance it's a bit confusing to create a new hook with that general name

Reply:
also usually we create a hook with chainId param but there you are iterating through all the chains, I'm not sure it may be needed in other cases if this data will be reused

It might make sense to call multiple hooks and wrap them in an aggregating hook to count the total values